core/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -86,6 +90,8 @@ public final class OpenTelemetryRumBuilder {
 
     private Resource resource;
 
+    private final ExecutorService executorService = Executors.newSingleThreadExecutor();
```

Comment:
Been a while since I've dealt with this but since the usage for this is for a single task, do we need to shut it down afterwards?

Reply:
super good call out, completely forgot that. Because theres nothing stopping build from getting repeat calls im going to bring this as a val into the function and shut it down after execution.  What do you think?

Reply:
Accidentally deleted the comment. Good callout! Ill instantiate and shut it down in the build call to keep the build call recallable